runtime/etc/Makefile.include

```diff
@@ -52,11 +52,36 @@ CHPL_RT_INC_DIR = $(RUNTIME_INCLS)
 
 CHPL_RT_LIB_DIR = $(CHPL_MAKE_HOME)/$(LIB_RT_DIR)
 
+# These Make ifdefs are used for printcompileline and printcflags
+# in order to correctly set the equivalent to COMP_GEN_CFLAGS.
+# In the future, we could simplify codegen_makefile if they
+# are always relied upon.
```

Comment:
Since we are waiting on this change, would you like to include the change to simplify codegen_makefile? I think it would be nice to use the same mechanism for both the C and LLVM backends.